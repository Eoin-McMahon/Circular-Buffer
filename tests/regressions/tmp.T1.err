FAIL -> Cannot add an element
FAIL -> Cannot remove an element
FAIL -> Dose not detect overflows.
FAIL -> Dose not detect overflows.
