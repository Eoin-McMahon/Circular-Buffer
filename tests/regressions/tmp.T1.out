 Inserted -> 1
 Inserted -> 1

 Queue is empty Inserted -> 1
 Inserted -> 1
 Inserted -> 1
 Inserted -> 1
 Inserted -> 1
Queue is full!!
 Queue is empty